curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md

```diff
@@ -0,0 +1,84 @@
+---
+id: 5d383f6f2cd33e8211fad377
+title: Enter in openCV
+challengeType: 0
+videoUrl: 
+---
+
+## Description
+<section id='description'>
+Welcome to freeCodeCamp's HTML coding challenges. These will walk you through web development step-by-step.
+First, you'll start by building a simple web page using HTML. You can edit <code>code</code> in your <code>code editor</code>, which is embedded into this web page.
+Do you see the code in your code editor that says <code>&#60;h1&#62;Hello&#60;/h1&#62;</code>? That's an HTML <code>element</code>.
+Most HTML elements have an <code>opening tag</code> and a <code>closing tag</code>.
+Opening tags look like this:
+<code>&#60;h1&#62;</code>
+Closing tags look like this:
+<code>&#60;/h1&#62;</code>
+The only difference between opening and closing tags is the forward slash after the opening bracket of a closing tag.
+Each challenge has tests you can run at any time by clicking the "Run tests" button. When you pass all tests, you'll be prompted to submit your solution and go to the next coding challenge.
+</section>
+
+## Instructions
+<section id='instructions'>
+To pass the test on this challenge, change your <code>h1</code> element's text to say "Hello World".
+</section>
+
+## Tests
+<section id='tests'>
+
+```yml
+tests:
+  - text: Your <code>h1</code> element should have the text "Hello World".
+    testString: assert.isTrue((/hello(\s)+world/gi).test($('h1').text()), 'Your <code>h1</code> element should have the text "Hello World".');
```

Comment:
It seems to me that this field contains JS code which tests user's code. Try to add `console.log(1)` here to see.